### PR TITLE
Added backup success check before restoring:PA-1762

### PR DESCRIPTION
**Pull request by snigdha-px**
**What this PR does / why we need it**:
Added backup success check before starting the restore
**Which issue(s) this PR fixes** (optional)
Closes #https://portworx.atlassian.net/browse/PA-1762

**Special notes for your reviewer**:

### CODE REVIEW

File: tests/backup/backup_namespace_labelled_test.go [hunk 1]
```diff
@@ -1657,6 +1657,8 @@ var _ = Describe("{NamespaceLabelledBackupOfEmptyNamespace}", func() {
 			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of next namespace labelled schedule backup %s with namespace included", scheduleBackupWithNamespace))
 			err = NamespaceLabelBackupSuccessCheck(scheduleBackupWithNamespace, ctx, bkpNamespaces, labelForNamespace)
 			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying if labelled namespace %s are included in this schedule backup %s ", bkpNamespaces, nextScheduleBackupName))
+			err = backupSuccessCheckWithValidation(ctx, scheduleBackupWithNamespace, scheduledAppContexts, orgID, maxWaitPeriodForBackupCompletionInMinutes*time.Minute, 30*time.Second)
```
Comment (kphalgun-px) [inline tests/backup/backup_namespace_labelled_test.go:1660]: ValidateBackup?

**Review by kphalgun-px — APPROVED** [review]
LGTM

File: tests/backup/backup_namespace_labelled_test.go (hunk 1 above)
Comment (snigdha-px) [reply tests/backup/backup_namespace_labelled_test.go:1660]: Discussed .

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

**Pull request merged**